browser/admin/src/integrator/WordBook.ts

```diff
@@ -204,7 +207,7 @@ class WordBook {
 	private createDropdownElement(className, textContent, parent) {
 		const element = document.createElement('div');
 		element.className = className;
-		element.textContent = textContent;
+		element.textContent = _(textContent);
```

Comment:
are you sure this is a localizable English word?

Reply:
No I'm not sure but saw similar pattern here: https://github.com/codewithvk/online/blob/master/browser/src/control/Control.Toolbar.js#L676

Reply:
you pass localized textContent (i.e. the opt.heading), so no extra _('') is required.